### DISCUSSION TITLE
Some Keyboard backlight optoins disappeared in v0.107

### DISCUSSION THREAD

**Discussion by martin6982** [seerge/g-helper#1033] [Q&A] [answered]
Hi @seerge ,I download the v0.107 version and found that my the aura-type options in keyboard backlight disppeared, please check out these 2 screenshots:
https://imgur.com/muKVvnX
https://imgur.com/h7yVfg0

And I also re-installed the v0.104~v0.106, this problem didn't appear, how can I fix this problem in v0.107?

**Comment by seerge** [comment, marked as answer]
@martin6982 hello, I have removed color options cause someone with GA503RM literally came and stated that colors are not supported on this model : https://github.com/seerge/g-helper/issues/1012 

Does your model supoprt RGB ? if So I can return colors :) 

[GHelper.zip](https://github.com/seerge/g-helper/files/12293041/GHelper.zip)

**Reply by martin6982** [reply]
Hi @seerge My model is GA503RM 2022(R7-6800HS+RTX3060), and yes my model has One-Zone RGB, and the options in the case #1012 and color pallate works well, do you need my endpoint code or anything else to let you check the difference between the model in the #1012  and mine?

**Reply by seerge** [reply]
No, i trust you. I will just restore RGB options in next release, so far you can use the one i linked

**Reply by MadOokami** [reply]
I have a G15 GA503RM and it definitely supports RGB
